addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -307,6 +299,7 @@ def test_in_invoice_line_onchange_product_2_with_fiscal_pos(self):
         })
 
         uom_dozen = self.env.ref('uom.product_uom_dozen')
+        self.env.user.groups_id += self.env.ref('uom.group_uom')
```

Comment:
https://github.com/odoo/odoo/pull/96134/files#diff-f319bb2adb371a83f9e28439c6b0ba53a6c6d38b15c1fdbe0f868e558dba2065L128 
:man_shrugging:

Reply:
I'm not sure the link sends me to the right place (always easier to have a link on the commit directly than on the PR)
But if you are telling me that the UoM can be set in the setUpClass in the `create`, it is not the same.
It is blocked from setting it in the view (with the `Form` emulator) because the field is invisible, see the commit I linked above)

Reply:
https://github.com/odoo-dev/odoo/blob/49821663bde497a0ea7314d2165ae5308a920e63/addons/account/tests/test_account_move_in_invoice.py#L129